base/src/main/java/proguard/evaluation/Variables.java

```diff
@@ -90,6 +92,7 @@ public void initialize(Variables other)
     {
         if (this.size < other.size)
         {
+            // TODO(MJ): more errors get thrown in this file, I think they could all be of our special type?
```

Comment:
To be resolved, just like the other TODOs. A special type may make sense if you intend to specifically catch it elsewhere.